src/main/webapp/app/shared/components/clone-repo-button/clone-repo-button.component.html

```diff
@@ -24,7 +24,7 @@
         <h5>{{ cloneHeadline | artemisTranslate }}</h5>
         <div class="d-flex" style="margin: 0; max-width: 100%">
             <div *ngIf="sshEnabled" aria-label="Button group with nested dropdown" class="btn-group" ngbDropdown role="group" style="margin-bottom: 10px">
```

Comment:
Was removing the manual height enough? The latest commit c606530ed1035f1b65e0c0938c4cc7f9adc31617 didn’t change anything else.

Reply:
Hm, indeed.